wro4j-core/src/main/java/ro/isdc/wro/model/factory/XmlModelFactory.java

```diff
@@ -359,17 +372,30 @@ private void parseResource(final Element resourceElement, final Collection<Resou
       resources.addAll(getResourcesForGroup(uri));
     }
     if (type != null) {
-      final String minimizeAsString = resourceElement.getAttribute(ATTR_MINIMIZE);
-      final boolean minimize = StringUtils.isEmpty(minimizeAsString) ? true
-          : Boolean.valueOf(resourceElement.getAttribute(ATTR_MINIMIZE));
-      final Resource resource = Resource.create(uri, type);
-      resource.setMinimize(minimize);
+      final Resource resource = createResource(resourceElement, type);
       LOG.debug("\t\tadding resource: {}", resource);
       resources.add(resource);
     }
   }
 
   /**
+   * Creates a resource from a given resourceElement. The element is guaranteed to be of a simple non-recursive type
+   * (i.e. css or js).
+   *
+   * @param resourceElement
+   *          Resource element to parse.
+   */
+  protected Resource createResource(final Element resourceElement, final ResourceType type) {

```

Comment:
I think the ResourceType is something that can be resolved from resourceElement. As result, the ResourceType parameter is redundant.